### DISCUSSION TITLE
Account Marked as flagged, how to resolve?

### DISCUSSION THREAD

**Discussion by zihanghliu** [community/community#82397] [Other Feature Feedback, Questions, & Ideas]
Hi there. Earlier today I found I lost access to the GitHub copilot functionalities. After I went to my GitHub profile I got a message saying my account had been flagged. I have been using my account normally and have done no actions that violate the rules. Has anyone encountered similar problems? I wish to resolve this issue as fast as possible, as I am currently applying for graduate programs and I have submitted some of my works in the form of GitHub repo links. I have already filed for a ticket to resolve this, but I wonder if there are faster ways to resolve this. Thank you very much!
<img width="999" alt="Screenshot 2023-12-27 at 4 09 53 PM" src="https://github.com/community/community/assets/92193637/a3488fe2-ff4d-4810-8e3b-720c098a3e17">

**Comment by MarcNarbon** [comment]
Did u have any success? I cant use my Copilot neither. My flag was raised because of the activity some hacker did on my account when it was compromised.
